fast-logger/System/Log/FastLogger.hs

```diff
@@ -1,7 +1,7 @@
 -- | This module provides a fast logging system which
 --   scales on multicore environments (i.e. +RTS -N\<x\>).
 {-# LANGUAGE CPP #-}
-{-# LANGUAGE Safe #-}
```

Comment:
Because GHC tell me i can't use unix package while mark module safe, because unix is not safe.